.eslintrc.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+	root: true,
+	extends: [
+		'kentcdodds',
+		'plugin:jest/recommended',
+		'plugin:jest-formatting/recommended',
+		'prettier',
+	],
+	rules: {
+		// Base
+		'max-lines-per-function': 'off',
+		'no-restricted-imports': [
+			'error',
+			{
+				patterns: ['@typescript-eslint/utils/dist/*'],
```

Comment:
I'm asking because I'm curious - why aren't we allowing importing from `@typescript-eslint/utils/dist/*`? Could we clarify that with the `message` option for this rule and in that message also guide people to importing from somewhere else (if such library or folder exists)?

Reply:
Good point! Not sure if still possible, but you could import typescript-eslint utils from the `dist/` folder instead of their exported objects. You can see [the original commit here](https://github.com/testing-library/eslint-plugin-testing-library/pull/385/commits/1b2024c8f892cc7b0a5a2b462f0946213edae867).

I'll add a message to clarify it must be just exported from `@typescript-eslint/utils` instead.

Reply:
Fixed! 
![imagen](https://user-images.githubusercontent.com/2677072/209098368-67e9685c-5e4d-40bb-aa09-9bf19d75eeb6.png)